src/main/java/uk/nhs/digital/nhsconnect/lab/results/inbound/queue/FhirDataToSend.java

```diff
@@ -2,16 +2,12 @@
 
 import lombok.Getter;
 import lombok.Setter;
-import org.hl7.fhir.dstu3.model.Parameters;
+import org.hl7.fhir.dstu3.model.Bundle;
 
 @Getter
 @Setter
 public class FhirDataToSend {
-    private Object content;
-    private String operationId;
 
-    public FhirDataToSend setContent(final Parameters parameters) {
-        this.content = parameters;
-        return this;
-    }
+    private Bundle content;
+    private String operationId;
```

Comment:
Would you say this is needed? I don't think we se the value

Reply:
I thought this class was maybe a copy+paste from NHAIS, but doesn't look like it is.
I see you originally added this, was there a reason why `operationId` was added? If not, we can remove this class entirely, and use `Bundle` instead of `FhirDataToSend`

Reply:
I thought i may have added it :D I would have taken it from one of the other integration adaptors. I can see the getter is used, but i don't think we're actually setting the `operationId`, so it'll always be null. 

I'll see if i can find any info on this

Reply:
Response on Teams - @bartek-sarul is removing the `operationId`

Reply:
Cheers for checking that, I've made that change on my PR as well to remove `FhirDataToSend` (can simply be replaced by `Bundle`). Merge conflicts (if any) shouldn't be difficult to resolve